graphql-kotlin-schema-generator/src/main/kotlin/com/expedia/graphql/generator/state/SchemaGeneratorState.kt

```diff
@@ -4,21 +4,24 @@ import com.expedia.graphql.directives.DeprecatedDirective
 import graphql.Directives
 import graphql.schema.GraphQLDirective
 import graphql.schema.GraphQLType
+import java.util.concurrent.ConcurrentHashMap
 
 internal class SchemaGeneratorState(supportedPackages: List<String>) {
     val cache = TypesCache(supportedPackages)
     val additionalTypes = mutableSetOf<GraphQLType>()
-    val directives = mutableSetOf<GraphQLDirective>()
+    val directives = ConcurrentHashMap<String, GraphQLDirective>()
```

Comment:
updated to `ConcurrentHashMap` instead of just simple `HashMap` as by default maven surefire plugin runs test in parallel which lead to some concurrency issues